src/components/Posts/PostItem.tsx

```diff
@@ -85,7 +85,7 @@ const PostItem = ({ post }: PostProps) => {
     <Card sx={styles.card}>
       <CardHeader
         avatar={<UserAvatar username={post.username} />}
-        action={<PostMenu authorId={post.userId} postId={post.postId} />}
+        action={<PostMenu authorId={post.userId} post={post} />}
```

Comment:
Similar comment as my initial comment about the redundancy here. Since we are already passing `post.userId` as part of the `post` object we don't need the `authorId` to be a separate prop here